src/python/pants/backend/jvm/targets/runtime_platform_mixin.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from abc import ABC
+
+from pants.backend.jvm.subsystems.jvm_platform import JvmPlatform
+from pants.base.payload_field import PrimitiveField
+
+
+class RuntimePlatformMixin(ABC):
+  """A mixin that identifies a target class as one that can have a jvm runtime_platform.
```

Comment:
Based on our discussion, I feel like this docstring should use the word "root" somewhere.

Reply:
Will do.